### PR TITLE
Add eslint-plugin-es5

### PR DESCRIPTION
**Pull request by nkt**

### CODE REVIEW

**Comment by ai** [comment]
If you have mode non-ES6 rules you can enable more rules :)

File: index.js [hunk 1]
```diff
@@ -18,7 +21,10 @@ module.exports = {
     'no-shadow': 'error',
     'max-len': [2, 80],
     'strict': ['error', 'global'],
-    'no-new': 'off'
+    'no-new': 'off',
+
+    'es5/no-block-scoping': 'error',
+    'es5/no-arrow-functions': 'error'
```
Comment (ai) [inline index.js]: Feel free to enable more rules, if you have them.

File: package.json [hunk 2]
```diff
@@ -13,12 +13,14 @@
   "peerDependencies": {
     "eslint": "^3.8.0",
     "eslint-config-standard": "^6.2.0",
+    "eslint-plugin-es5": "^0.2.0",
```
Comment (ai) [inline package.json]: Why you use `0.2` version if you already released `0.3`?

**Review by ai — CHANGES_REQUESTED** [review]

**Comment by nkt** [comment]
@ai plugin have been unfinished when I sent this PR. But now it's good enough for production usage.
PR updated, please review it.

**Review by ai — APPROVED** [review]

**Comment by ai** [comment]
@nkt thanks for your work. I will promote it tomorrow or after-tomorrow. Anyway internet is not active right now :).

**Pull request merged**